### PR TITLE
NIAD - 761: Fix object mapper error and add component to executors

### PR DESCRIPTION
**Pull request by Broom94**

### CODE REVIEW

**Review by mmackiewicz1 — DISMISSED** [review]

File: service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskDefinition.java [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
 import uk.nhs.adaptors.gp2gp.common.task.TaskDefinition;
 import uk.nhs.adaptors.gp2gp.common.task.TaskType;
 
+@Data
+@Jacksonized
```
Comment (mattd-kainos) [inline service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskDefinition.java:12]: Does it work using only the Jacksonzied annotation? Lombok's documentation seems to indicate that it should. https://projectlombok.org/features/experimental/Jacksonized

If Data is required remove the now redundant Getter annotation

File: service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskDefinition.java [hunk 2]
```diff
@@ -1,11 +1,14 @@
 package uk.nhs.adaptors.gp2gp.gpc;
 
+import lombok.Data;
```
Comment (michael-kainos) [inline service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskDefinition.java:3]: checkstyle is failing the build that this import is no longer used. Please remove

File: service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcStructuredTaskDefinition.java [hunk 3]
```diff
@@ -1,11 +1,14 @@
 package uk.nhs.adaptors.gp2gp.gpc;
 
+import lombok.Data;
```
Comment (michael-kainos) [inline service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcStructuredTaskDefinition.java:3]: Same here, remove as this is failing on checkstyle

**Review by michael-kainos — CHANGES_REQUESTED** [review]
Remove imports no longer being used, gradle build is currently failing

**Comment by mmackiewicz1** [comment]
Please update your branch with the changes present on the main one.

**Review by mmackiewicz1 — APPROVED** [review]

**Review by michael-kainos — APPROVED** [review]

**Pull request merged**